### ISSUE TITLE
Type 'SRT' does not conform to protocol 'Hashable'

### ISSUE THREAD

**Issue by Cardano-DoubleMe-Me** [schwa/SwiftGLTF#4]
When I build Demo, see this error
<img width="1139" alt="스크린샷 2023-07-11 오후 12 32 43" src="https://github.com/schwa/SwiftGLTF/assets/124238079/43c619a1-82a1-481f-b09a-216460e11c41">

**Comment by schwa** [comment]
One of the dependencies changed and it wasn't used a pinned Package file. I'll look into it.

**Comment by schwa** [comment]
I'm not able to reproduce at all actually. I can build just fine from main and If I update dependencies I can still build. What version you building and what Xcode version?

**Comment by Cardano-DoubleMe-Me** [comment]
I use Xcode 14.3.1.

**Comment by Cardano-DoubleMe-Me** [comment]
Umm, maybe `SIMDSupport` library need fixing. `SRT` have not `hash` method 
<img width="849" alt="스크린샷 2023-07-12 오전 9 51 44" src="https://github.com/schwa/SwiftGLTF/assets/124238079/1728d27b-762c-48ae-bdfb-c75f11aaf5cb">

**Comment by Cardano-DoubleMe-Me** [comment]
```
public struct SRT: Hashable {
    public var scale: SIMD3<Float> = .unit
    public var rotation: simd_quatf = .identity
    public var translation: SIMD3<Float> = .zero

    public init(scale: SIMD3<Float> = .unit, rotation: simd_quatf = .identity, translation: SIMD3<Float> = .zero) {
        self.scale = scale
        self.rotation = rotation
        self.translation = translation
    }

    public var matrix: simd_float4x4 {
        let scaleMatrix = simd_float4x4(scale: scale)
        let rotationMatrix = simd_float4x4(rotation)
        let translationMatrix = simd_float4x4(translate: translation)
        return translationMatrix * rotationMatrix * scaleMatrix
    }
    
    public func hash(into hasher: inout Hasher) {
        // Your code
    }
}

```

**Comment by schwa** [comment]
This should be solved by tag/release 0.0.4 - you're going to want to pin your Package.swift to NOT use  anything newer as 0.0.4 is the last release I'll be supporting an older SDK.
